system/jlib/jtrace.cpp

```diff
@@ -808,8 +865,11 @@ class CSpan : public CInterfaceOf<ISpan>
 
     ISpan * createClientSpan(const char * name, const SpanTimeStamp * spanStartTimeStamp = nullptr) override;
     ISpan * createInternalSpan(const char * name, const SpanTimeStamp * spanStartTimeStamp = nullptr) override;
+    ISpan * createConditionalInternalSpan(const char * name, stat_type thresholdMs) override;
```

Comment:
Conditionally suppressing spans seems to only make sense for child spans, however there's concern about dropping client spans which might have decimated traceparent information to its target server.